app/Actions/Site/Deploy.php

```
@@ -54,10 +57,23 @@ public function run(Site $site, bool $modern = true): Deployment
+    private function broadcastDeploymentCreated(Site $site, Deployment $deployment): void
+    {
+        $deployment->loadMissing('log');
+
+        SocketEvent::dispatch(new SocketEventDTO(
+            projectId: $site->server->project_id,
+            type: 'deployment.created',
+            data: new DeploymentResource($deployment),
+        ));
+    }
+
     private function deployClassic(Site $site, Deployment $deployment, ServerLog $log): Deployment
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$log'. (undefined)

(UnusedFormalParameter)